### DISCUSSION TITLE
Engram layout on Android with Multiling O Keyboard

### DISCUSSION THREAD

**Discussion by sunaku** [binarybottle/engram-layout#36] [Show and tell]
Hey Arno and community,

I've been using Engram on my Android smartphone through the [Multiling O Keyboard](https://play.google.com/store/apps/details?id=kl.ime.oh)'s DIY layouts feature.  The best part is that it supports gesture typing, which I find essential for typing without looking at the on-screen keyboard on the non-tactile glass screen.

To try this out yourself, copy the layout source code below and then navigate to Settings > Layouts > DIY, then drag from Spacebar to Layouts, then select `[+ DIY ]`, and finally paste from clipboard.  It should take effect immediately and be saved as "Engram" layout. :nerd_face:

Or, to look before you leap, check out the screenshots below.

Cheers.

## Source code

```
OK_Layout_Begin
Name:Engram
1234567890
byouldwvz
cieahtsnq
gxjkrmfp[EMOJI]
|=~+<>^&%*
BYOULDWVZ
CIEAHTSNQ
GXJKRMFP[EMOJI]
|=~+<>^&%*
()•'"…#@/
{};,.:$`\
[]_-?!€£¥
OK_Layout_End
```

### Lowercase (shift OFF)
![lowercase - shift OFF](https://user-images.githubusercontent.com/9863/145528443-a3917cdd-6c95-40e2-b905-5366fdf52ef8.png)

### Uppercase (shift ON)
![uppercase - shift ON](https://user-images.githubusercontent.com/9863/145528564-3f853b0e-7834-4e09-a3ae-7f10e94afb65.png)

**Comment by binarybottle** [comment]
Amazing!  THANK YOU, @sunaku!  
I look forward to trying it out on my Android phone!

**Comment by binarybottle** [comment]
Works like a charm -- thank you, @sunaku!

**Comment by binarybottle** [comment]
![Screenshot_20211218-165145](https://user-images.githubusercontent.com/236532/146656494-30467a38-b0d4-46ce-a189-f0e040c2fa1b.png)
